### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by lnmunhoz** [whitecolor/ts-node-dev#190]
Hi, I am using this package daily and seems like a lot of another people as well.

Would you like to setup semantic versioning? I can set it up and send a PR if you're interested. I normally use [release-it](https://github.com/release-it/release-it) package that is pretty simple to setup.

Let me know!

**Comment by whitecolor** [comment]
Yes, it would be nice to have normal release cycle for the project. How `release-it` is different from `semantic-release`?

**Comment by lnmunhoz** [comment]
They have the same purpose, but after trying `semantic-release` for a while and moving to `release-it`, I found `release-it` much more straightforward and easier to setup/maintain. 

I will make a PR sometime this week so you can try it out yourself.
